<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>904</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Momdis : projectile nucleon knockout reaction in eikonal approximation (Ref: C. Bertulani.... )</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_32">
     <property name="text">
      <string>T(P,C)X reaction P=C+nucleon</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>base</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>ZP</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="lineEdit_ZP">
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLineEdit" name="lineEdit_AP">
        <property name="text">
         <string>15</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>AP</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="lineEdit_ZT">
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>ZT</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Target</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>AT</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_30">
        <property name="text">
         <string>Projectile</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QLineEdit" name="lineEdit_AT">
        <property name="text">
         <string>9</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>E/A (MeV)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="lineEdit_ECA">
        <property name="text">
         <string>103.0</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="label_31">
        <property name="text">
         <string>valence particle</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QComboBox" name="comboBox_nucleon">
        <item>
         <property name="text">
          <string>neutron</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>proton</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>S.F.</string>
        </property>
       </widget>
      </item>
      <item row="0" column="8">
       <widget class="QLineEdit" name="lineEdit_S2C">
        <property name="text">
         <string>1.0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Projectile Bound state</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>N0</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>L0</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_N0">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="lineEdit_L0">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLineEdit" name="lineEdit_J0">
        <property name="text">
         <string>0.5</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>J0</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>AA</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>V0</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLineEdit" name="lineEdit_AA">
        <property name="text">
         <string>0.6</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="lineEdit_R0">
        <property name="text">
         <string>2.67</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>R0</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>VS0</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_V0">
        <property name="text">
         <string>-61.849</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_VS0">
        <property name="text">
         <string>0.0</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>RS0</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLineEdit" name="lineEdit_RS0">
        <property name="text">
         <string>2.4</string>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>RC</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>AAS</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QLineEdit" name="lineEdit_AAS">
        <property name="text">
         <string>0.6</string>
        </property>
       </widget>
      </item>
      <item row="1" column="7">
       <widget class="QLineEdit" name="lineEdit_RC">
        <property name="text">
         <string>2.67</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>t-rho-rho scattering</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_trhorho">
      <item row="2" column="3">
       <widget class="QComboBox" name="comboBox_den_c"/>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_22">
        <property name="text">
         <string>core density</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QComboBox" name="comboBox_IPAULIn">
        <item>
         <property name="text">
          <string>0(no correction in nn-cross section)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1(correction in nn-cross section)</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string>Target density</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>nucleon density</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>IPAULI</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_26">
        <property name="text">
         <string>core-target scattering</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBox_den_n"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_25">
        <property name="text">
         <string>nucleon-target scattering</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_23">
        <property name="text">
         <string>IPAULI</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QComboBox" name="comboBox_IPAULIc">
        <item>
         <property name="text">
          <string>0(no correction in nn-cross section)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1(correction in nn-cross section)</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QComboBox" name="comboBox_den_t"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Work</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_24">
        <property name="text">
         <string>compute</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_sigma_type"/>
      </item>
      <item>
       <widget class="QLabel" name="label_27">
        <property name="text">
         <string>L0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_sig_L0">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_28">
        <property name="text">
         <string>M0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_sig_M0">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
